package.json

```
@@ -30,17 +43,24 @@
   "author": "",
   "license": "Apache-2.0",
   "dependencies": {
-    "lit": "^3.1.0",
-    "jwt-decode": "^3.0.0"
+    "jwt-decode": "^3.0.0",
+    "lit": "^3.1.0"
```

I'd lean towards keeping the range wide for more de-duplication opportunities
```suggestion
    "jwt-decode": "^3.0.0",
    "lit": "^3.0.0"
```